OpenRA.Mods.Common/Traits/World/EditorActorPreview.cs

```diff
@@ -98,6 +99,8 @@ public EditorActorPreview(WorldRenderer worldRenderer, string id, ActorReference
 
 			SelectionBox = new SelectionBoxAnnotationRenderable(new WPos(CenterPosition.X, CenterPosition.Y, 8192),
 				new Rectangle(Bounds.X, Bounds.Y, Bounds.Width, Bounds.Height), Color.White);
+
+			onCellEntryChanged = (CPos pos) => GeneratePreviews();
```

Comment:
```suggestion
			onCellEntryChanged = _ => GeneratePreviews();
```

this is the way we do it. It's also a bit weird that ramp changes trigger height cell map. That will need to be fixed in the future.

This also updates all previews in the map editor. That may be a bit extreme, though I doubt it costs enough to care

Reply:
other approach I've tested was not caching the ramp values and just getting them on each get orientation call, didn't seem to affect perf at all even in map full of actors but I suppose it's not the right approach